app/web/src/components/FuncEditor/FuncList.vue

```diff
@@ -1,17 +1,14 @@
 <template>
   <div class="flex flex-col overflow-hidden h-full relative">
     <TreeNode
-      v-for="(label, kind, index) in CUSTOMIZABLE_FUNC_TYPES"
+      v-for="(label, kind) in CUSTOMIZABLE_FUNC_TYPES"
       :key="kind"
       enableDefaultHoverClasses
       enableGroupToggle
       alwaysShowArrow
       indentationSize="none"
       leftBorderSize="none"
-      :defaultOpen="
-        (defaultOpen || (index === 0 && firstOpen)) &&
-        !(funcsForKind(kind).length <= 0)
-      "
+      defaultOpen
```

Comment:
Do we not want func types which have zero children to be collapsed by default anymore?

Reply:
So.. thats less the issue than `defaultOpen` can't really be dynamic... because its the default... if the prop changes to `true` it doesn't open the children... which is sad ;(

Reply:
Ah I see! Well there's probably a better way to do this behavior anyway.